recipes/SimpleElastix/bld.bat

```diff
@@ -0,0 +1,4 @@
+
+
+
+if errorlevel 1 exit 1
```

Comment:
I definitely would like to build on Windows. I have no idea what to put here yet, though :)